docs/tasks/issuers/setup-acme/dns01/cloudflare.rst

```diff
@@ -2,6 +2,10 @@
 Cloudflare
 =========================
 
+There are two ways to authenticate with Cloudflare. If you want to use
+API key you must have apiKeySecretRef defined, if you want to use API token
```

Comment:
Can you separate the two auth methods under separate headers?

It would also be nice to make it error if both are defined too I think
@munnerz